app/src/main/java/com/example/myapplication/RegisterFragment.kt

```diff
@@ -60,10 +63,10 @@ class RegisterFragment : Fragment() {
 
         binding.buttonRegister.setOnClickListener {
             lifecycleScope.launch {
-                _passwordless.register(null, null)
+                val httpClient = YourBackendHttpClientFactory.create("https://demo-backend.passwordless.dev/")
+                val responseToken = httpClient.register("Shubham"+ Random.nextDouble()).body()?.token!!
```

Comment:
We can take the input from a text field here.

Reply:
I have put that in todo, as that would add extra steps while testing.